package.json

```diff
@@ -154,6 +154,16 @@
                         "replace",
                         "allowDuplicates"
                     ]
+                },
+                "numberedBookmarks.fillColor": {
```

Comment:
Rename the setting to `gutterIconFillColor` to be more clear that this _fillColor_ is related to the gutter icon. It also be similar to the other `gutterIconPath` from my other Bookmarks extension [link](
https://github.com/alefragnani/vscode-bookmarks/#available-settings)